apache/files/_log.conf

```diff
@@ -1,5 +1,10 @@
+{%- set error_log = site.get('log', {}).get('error', {}) %}
+{%- set custom_log = site.get('log', {}).get('custom', {}) %}
 
-	LogLevel warn
-
-	ErrorLog /var/log/apache2/{{ site_name }}.error.log
-	CustomLog /var/log/apache2/{{ site_name }}.access.log vhost_combined
+{%- if error_log.get('enabled', True) %}
+	LogLevel {{ error_log.level|default('warn') }}
+	ErrorLog {{ error_log.file|default('/var/log/apache2/' ~ site_name ~ '.error.log') }}
+{%- endif %}
+{%- if custom_log.get('enabled', True) %}
+	CustomLog {{ custom_log.file|default('/var/log/apache2/' ~ site_name ~ '.access.log') }} "{{ custom_log.format|default('vhost_combined') }}"
```

Comment:
You should maybe take log directory path from Jinja, because AFAIK on RHEL it's /var/log/httpd or something?
Also please retab this file to have 4 spaces instead of one tab.